### PR TITLE
feat(ui5-date-*): adapt date and time controls to timezone feature

### PR DESCRIPTION
**Pull request by tsanislavgatev**
Adapting the date and time controls to the timezone feature, allowing the controls to present date in different timezones.

Adaptation is done by using the UI5Date object provided by ui5.

Can be tested either via  html tag:

```html
<script data-ui5-config type="application/json">
	{
		"language": "EN",
		"timezone" : "Pacific/Apia"
	}
</script>
```
or by adding a query parameter to the url: `?sap-ui-timezone=XXXX`

The timezone identifiers format is the one used in IANA timezone database.

### CODE REVIEW

File: packages/website/docs/_components_pages/main/TimePicker.mdx [hunk 1]
```diff
@@ -22,4 +23,8 @@ import States from "../../_samples/main/TimePicker/States/States.md";
 ### States
 TimePicker supports several semantic value states, readonly, disabled, etc.
 
-<States />
\ No newline at end of file
+<States />
+
+### Timezones
+You can set to the configuration the preferred timezone, such as: Asia/Tokyo, Pacific/Apia, Asia/Kolkata, Europe/Sofia and etc.
```
Comment (BorisDafov) [inline packages/website/docs/_components_pages/main/TimePicker.mdx:29]: time zone

File: packages/website/docs/_samples/main/Calendar/CalendarInDifferentTimezone/sample.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<!-- playground-fold -->
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Calendar in different time zone</title>
```
Comment (BorisDafov) [inline packages/website/docs/_samples/main/Calendar/CalendarInDifferentTimezone/sample.html:8]: Title case:
Calendar in Different Time Zone

File: packages/website/docs/_samples/main/DatePicker/DatePickerInDifferentTimezone/sample.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<!-- playground-fold -->
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>DatePicker in different time zone</title>
```
Comment (BorisDafov) [inline packages/website/docs/_samples/main/DatePicker/DatePickerInDifferentTimezone/sample.html:8]: DatePicker in Different Time Zone

File: packages/website/docs/_samples/main/DateTimePicker/DateTimePickerInDifferentTimezone/sample.html [hunk 4]
```diff
@@ -0,0 +1,28 @@
+<!-- playground-fold -->
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>DateTimePicker in different time zone</title>
```
Comment (BorisDafov) [inline packages/website/docs/_samples/main/DateTimePicker/DateTimePickerInDifferentTimezone/sample.html:8]: DateTimePicker in Different Time Zone

File: packages/website/docs/_samples/main/TimePicker/TimePickerInDifferentTimezone/sample.html [hunk 5]
```diff
@@ -0,0 +1,28 @@
+<!-- playground-fold -->
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TimePicker om different time zone</title>
```
Comment (BorisDafov) [inline packages/website/docs/_samples/main/TimePicker/TimePickerInDifferentTimezone/sample.html:8]: TimePicker in Different Time Zone

**Review by BorisDafov — CHANGES_REQUESTED** [review]

**Comment by tsanislavgatev** [comment]
Mistakenly merged main and not release-2.0. The work will continue in: https://github.com/SAP/ui5-webcomponents/pull/8610

**Pull request closed without merging**